INSTALL.rst

```diff
@@ -217,7 +217,7 @@ libSplash
 HDF5
 """"
 - 1.8.6+
-- standard shared version (no c++, enable parallel)
+- standard shared version (no C++, enable parallelism)
```

Comment:
not in this context, since "parallel" is a keyword.
change to `enable parallel IO` please